package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "inky",
-  "version": "1.3.7",
+  "version": "1.3.8",
```

Comment:
Please do not change this.

Reply:
@DanielRuf thanks for the quick response, I've removed the change.